### DISCUSSION TITLE
Plugin that resets skills back to 1 after reaching 99 (with cycles)

### DISCUSSION THREAD

**Discussion by guy-jin** [runelite/runelite#18218] [Ideas]
Similar to Prestige plugin but instead of resetting at 92, it would do so at 99, and everytime you hit 99 after. Leveling from 1 to 99 would be on the normal rate. The plugin would then have an option to show on the skills tab for each skill which cycle of 99 leveling you are on by dividing your total experience in the skill with the 99 level requirement of 13,03m, with there being 15 cycles before you hit the 200m limit, at which point the reseting would stop.

The benefit would be to motivate players to keep on skilling past 99. Virtual levels are cool but when each virtual level gap is multiple millions of exp it becomes a slog. Instead of that, it would be more interactive to do it in level 99 cycles.

**Comment by cdfisher** [comment]
This would probably be more appropriate as a suggestion for the Prestige plugin. You can make that suggestion to the author [here](https://github.com/ckosidow/prestige-levelling/issues).
